### DISCUSSION TITLE
windows 64位版本, 打开失败啊, 情况如下图. 咋弄啊?

### DISCUSSION THREAD

**Discussion by sirblangch** [SPlayer-Dev/SPlayer#201] [Q&A] [answered]
![image](https://github.com/imsyy/SPlayer/assets/167659962/17e408d5-d115-4649-bebc-87a4f8717a24)

**Comment by FrzMtrsprt** [comment, marked as answer]
端口被占用了，重启一下电脑应该就释放了，如果还不行需要用命令行看是哪个软件占用的
